README.md

```diff
@@ -31,6 +31,41 @@ To sort or remove unused imports you can trigger the `LSP-typescript: Organize I
 
 The `LSP-typescript: Find Callers` command can be used to find what is calling the given symbol. It has some overlap with the built-in `LSP: Find References` command but returns only the places where the symbol was called.
 
+
+## Inlay hints (experimental)
+
+Inlay hints are short textual annotations that show parameter names, type hints.
+
+![inlay-hints](/images/inlay-hints.png)
+
+Inlay hints are disabled by default.
+To enable them, modify the following settings in `LSP-typescript.sublime-settings`:
+
+```js
+{
+  "settings": {
+    // Javascript inlay hints options.
+    "javascript.inlayHints.includeInlayEnumMemberValueHints": false,
+    "javascript.inlayHints.includeInlayFunctionLikeReturnTypeHints": false,
+    "javascript.inlayHints.includeInlayFunctionParameterTypeHints": false,
+    "javascript.inlayHints.includeInlayParameterNameHints": "none",
+    "javascript.inlayHints.includeInlayParameterNameHintsWhenArgumentMatchesName": false,
+    "javascript.inlayHints.includeInlayPropertyDeclarationTypeHints": false,
+    "javascript.inlayHints.includeInlayVariableTypeHints": false,
+    // Typescript inlay hints options.
+    "typescript.inlayHints.includeInlayEnumMemberValueHints": false,
+    "typescript.inlayHints.includeInlayFunctionLikeReturnTypeHints": false,
+    "typescript.inlayHints.includeInlayFunctionParameterTypeHints": false,
+    "typescript.inlayHints.includeInlayParameterNameHints": "none",
+    "typescript.inlayHints.includeInlayParameterNameHintsWhenArgumentMatchesName": false,
+    "typescript.inlayHints.includeInlayPropertyDeclarationTypeHints": false,
+    "typescript.inlayHints.includeInlayVariableTypeHints": false,
+  }
+}
+```
+
+Note: Inlay hints require TypeScript 4.4+ in the workspace.
```

Comment:
Maybe drop "in the workspace" since the package will come with TS 4.4+ so it should work even when it's not in the workspace.